controllers/ie_controller.go

```diff
@@ -0,0 +1,62 @@
+package controllers
+
+import (
+	"net/http"
+	"reflect"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/gin-gonic/gin"
+)
+
+// IEController Wrapped for Patient Collection
```

Comment:
It looks like you made this controller generic (great!) -- but the comments are still specific to the Patient controller.